docs/reference/packages.rst

```diff
@@ -31,6 +31,11 @@ written as follows::
 Other examples of package files can be found in the ``libs`` directory
 of the main Idris repository, and in `third-party libraries <https://github.com/idris-lang/Idris-dev/wiki/Libraries>`_.
 
+If you wish to comment packages, Idris style single line (``--``) and
+multiline comments (``{- -}``) can be used.  However, the file must
+start with the package delcaration ``package <name>``.
```

Comment:
no worries, that part is going to disappear once the 'block' PR has been merged, it makes the paragraph redundant.